### DISCUSSION TITLE
No connection after fresh install

### DISCUSSION THREAD

**Discussion by el-nino22** [telekom-security/tpotce#1679] [Q&A]
Hi everyone

I try to install t-pod on Ubuntu in proxmox but without success.

I installed Ubuntu 22.04.01 LTS with the minimal settings. Then I ran:
env bash -c "$(curl -sL https://github.com/telekom-security/tpotce/raw/master/install.sh)"

After it all ran through, I am able to connect via SSH on port 64295. Now I should reboot according to the guide.

After the reboot, I can connect via SSH again, but after a few seconds I loose connection. The system is still up and running and I can login via the proxmox console, but not via SSH and also there is nothing on the webgui. The same thing if I just ping the system. I can ping during startup but then I loose connection.

Anyone can help me on this issue?

**Comment by el-nino22** [comment]
I noticed that I should probably use Ubuntu **24**.04.01, so I did a fresh install but still have the same issue.

**Reply by Cyber-JL** [reply]
Im having the exact same issue

**Comment by Cyber-JL** [comment]
I had the same issue when doing the standard Hive install. The way I got around that was to take all honey pots off the hive and add a second system to be a sensor, essentially doing a distributed install.
